### PR TITLE
add FAN FAQ

### PR DESCRIPTION
**Pull request by rosemarymarano**

### CODE REVIEW

File: docs-source/content/faq/fan.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+title: "Disabling Fast Application Notifications"
+date:
```
Comment (markxnelson) [inline docs-source/content/faq/fan.md]: needs a date

File: docs-source/content/faq/fan.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+oracle.simplefan.impl.FanManager configure
+SEVERE: attempt to configure ONS in FanManager failed with oracle.ons.NoServersAvailable: Subscription time out
+```
+
+To correct the problem, you must disable FAN, in one of two places:
+
+1)	Through a system property at the domain, cluster, or server level.  
+
+To do this, edit the Domain Custom Resource to set the system property `oracle.jdbc.fanEnabled`
+to false as shown in the following example:
```
Comment (markxnelson) [inline docs-source/content/faq/fan.md]: put false in backticks

File: docs-source/content/faq/fan.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+---
+title: "Disabling Fast Application Notifications"
+date:
+draft: false
+---
+
+To support Fast Application Notifications (FAN), Oracle databases configure GRID.
```
Comment (markxnelson) [inline docs-source/content/faq/fan.md]: is GRID an acronym, we should expand it first time

File: docs-source/content/faq/fan.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+       set('Value', stbUser)
+       ls()
+       cd('../..')
+       ls()
+       create('oracle.jdbc.fanEnabled','Property')
+       ls()
+       cd('Property/oracle.jdbc.fanEnabled')
+       set('Value', 'false')
+       ls()
+```
```
Comment (markxnelson) [inline docs-source/content/faq/fan.md]: probably need an outro here .... 
maybe say restart the domain after making these changes or something?

File: docs-source/content/faq/fan.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  serverPod:
+    # an (optional) list of environment variable to be set on the servers
+    env:
+    - name: JAVA_OPTIONS
+      value: "-Dweblogic.StdoutDebugEnabled=false -Doracle.jdbc.fanEnabled=false"
+```
+
+2) Configure the data source connection pool properties.  
+
+The following WLST script creates a data source with the connection property set.
```
[the remark this answers is not in the document]
Comment (rosemarymarano) [reply docs-source/content/faq/fan.md]: Is this better, "The following WLST script adds the `oracle.jdbc.fanEnabled` property, set to `false`, to an existing data source.

File: docs-source/content/faq/fan.md (hunk 1 above)
Comment (mriccell) [reply docs-source/content/faq/fan.md]: October 11, 2019

File: docs-source/content/faq/fan.md (hunk 3 above)
Comment (mriccell) [reply docs-source/content/faq/fan.md]: It is not an acronym the product is actually called "Oracle Grid Infrastructure".

File: docs-source/content/faq/fan.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
+---
+title: "Disabling Fast Application Notifications"
+date:
+draft: false
+---
+
+To support Fast Application Notifications (FAN), Oracle databases configure GRID.
+GRID is typically associated with (and required by) Oracle RAC databases but can
+also be used in other configurations.  ATP-S does not provide GRID.
```
[the remark this answers is not in the document]
Comment (mriccell) [reply docs-source/content/faq/fan.md]: Oracle Autonomous Database: Dedicated vs Serverless.  ATP-S stands for ATP-Serverless

File: docs-source/content/faq/fan.md (hunk 3 above)
Comment (rosemarymarano) [reply docs-source/content/faq/fan.md]: I'll write that out.

File: docs-source/content/faq/fan.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (mriccell) [reply docs-source/content/faq/fan.md]: Yes

File: docs-source/content/faq/fan.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+When connecting to a database that does not have GRID, the only type of WebLogic Server
+supported data source is Generic Data Sources. Multi Data Sources and Active GridLink
+data sources cannot be used because they work with RAC.
+
+
+When connecting with a 12.2 driver to a database that does not have GRID, you will
```
Comment (mriccell) [inline docs-source/content/faq/fan.md]: WebLogic Server 12.2.1.3.0 shipped with the 12.2.0.1 Oracle driver, when connecting with this driver ...

File: docs-source/content/faq/fan.md (hunk 4 above)
Comment (mriccell) [reply docs-source/content/faq/fan.md]: After changing the Data Source Connection Pool configuration make sure to  undeployed or redeployed the Data Source or restart the WebLogic server for the attribute to take effect.

**Pull request merged**